### PR TITLE
Gulp build system

### PR DESCRIPTION
**Pull request by lossen**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    PROD = false;
+
+var gulp        = require('gulp'),
+    plumber     = require('gulp-plumber'),
+    size        = require('gulp-filesize'),
+
+    spritesmith = require('gulp.spritesmith'),
+    imagemin    = require('gulp-imagemin'),
+    pngquant    = require('imagemin-pngquant'),
+	buffer      = require('vinyl-buffer'),
```
Comment (tiero) [inline gulpfile.js]: bad alignment

File: gulpfile.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
+/*global require*/
+
+var WORK_OUT_FOLDER = './src/',
+    PROD = false;
```
Comment (tiero) [inline gulpfile.js]: require this constants/variables from a config file, let's put in the root folder a json called `config.json` and require from

File: package.json [hunk 3]
```diff
@@ -19,12 +13,31 @@
   "homepage": "https://bitbucket.org/vulpemventures/vulpem.com#readme",
   "devDependencies": {
     "@node-sass/node-module-importer": "^1.2.3",
+    "bulma": "^0.7.1",
     "http-server": "^0.11.1",
     "node-sass": "^4.9.2",
-    "nodemon": "^1.18.1",
+    "nodemon": "^1.18.3",
```
Comment (tiero) [inline package.json]: if not needed anymore, remove old dependencies used by node scripts, like http-server, node-sass, nodemon, parcel-bundler.

Please leave there `push-dir` used for deploy

File: scripts/deploy-gh-pages [hunk 4]
```diff
@@ -1,3 +0,0 @@
-#!/bin/bash
-sh ./scripts/bundle
-node_modules/.bin/push-dir --dir=dist --branch=gh-pages
```
Comment (tiero) [inline scripts/deploy-gh-pages]: this command will still be useful once the gulp will be able to create a production build in a folder called `dist`, you can re-use it.

File: gulpfile.js (hunk 1 above)
Comment (lossen) [reply gulpfile.js]: fixed

File: gulpfile.js (hunk 2 above)
Comment (lossen) [reply gulpfile.js]: I created the config.js file in the root folder.
The json format for these purposes is not very suitable in my opinion.

File: package.json (hunk 3 above)
Comment (lossen) [reply package.json]: fixed

File: gulpfile.js (hunk 2 above)
Comment (lossen) [reply gulpfile.js]: fixed

**Pull request merged**